picoquic/logwriter.c

```diff
@@ -634,7 +634,7 @@ void binlog_close_connection(picoquic_cnx_t * cnx)
     bytestream_buf stream_msg;
     bytestream * msg = bytestream_buf_init(&stream_msg, BYTESTREAM_MAX_BUFFER_SIZE);
     bytewrite_cid(msg, &cnx->initial_cnxid);
-    bytewrite_vint(msg, picoquic_current_time());
```

Comment:
Unfortunately, I had to revert this to 0. Otherwise the binary file comparison would complain about changes in the file. This isn't used right now in the converter, so it doesn't hurt.

I couldn't find a way to do this right (i.e. using simulated time during tests), so this is 0 for now.

Reply:
You want to call:
```
uint64_t picoquic_get_quic_time(picoquic_quic_t* quic); /* connection time, compatible with simulations */
```
That's will be either the simulated time during tests, or the wall time in production.

Reply:
Works now.